x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/index_pattern_prompt/index_pattern_prompt.tsx

```diff
@@ -30,20 +31,20 @@ export const IndexPatternPrompt: FC<Props> = ({ destIndex }) => {
 
   return (
     <>
-      <EuiText size="xs" color="warning">
+      <EuiText size="xs" color={color ?? 'warning'}>
         <FormattedMessage
           id="xpack.ml.dataframe.analytics.dataViewPromptMessage"
-          defaultMessage="No data view exists for index {destIndex}. "
+          defaultMessage="No data view exists for index{destIndex}. "
```

Comment:
Is there supposed to be a space in between for `index{destIndex}`

Reply:
I'm assuming it's like this for when destIndex is undefined. What about when destIndex is defined? It would read "No data view exists for indexmy-index-name. "

Reply:
Good catch! Updated in 7e8031b7f25662c8389e409f70fc8b762ff6fd8e